packages/p2p-v2/src/pages/my-ads/screens/MyAds/MyAdsTable/MyAdsTable.tsx

```diff
@@ -1,11 +1,13 @@
 import React, { memo } from 'react';
 import { p2p } from '@deriv/api';
-import { Loader } from '@deriv-com/ui';
+import { Button, Loader } from '@deriv-com/ui';
 import { Table } from '@/components';
+import { useDevice } from '@/hooks';
 import { MyAdsTableRow } from '../MyAdsTableRow';
+import { MyAdsToggle } from '../MyAdsToggle';
 import './MyAdsTable.scss';
 
-export type TMyAdsTableRowRendererProps = Partial<
+export type TMyAdsTableRowRendererProps = Required<
     NonNullable<ReturnType<typeof p2p.advertiserAdverts.useGet>['data']>[0]
 > & {
```

Comment:
@nada-deriv, do we need the `Required` if we have `NonNullable`? 🤔 

```suggestion
export type TMyAdsTableRowRendererProps = Required<
    NonNullable<ReturnType<typeof p2p.advertiserAdverts.useGet>['data']>[0]
> & {
```

Reply:
nonnullable checks for the `ReturnType<typeof p2p.advertiserAdverts.useGet>['data']` but requried is for the 0th element.

Reply:
Oooh, Alright